server/rest/tale.py

```diff
@@ -397,6 +398,18 @@ def updateTaleAccess(self, tale, access, publicFlags, public):
         return self._model.setAccessList(
             tale, access, save=True, user=user, setPublic=public, publicFlags=publicFlags)
 
+    @access.user(scope=TokenScope.DATA_READ)
+    @autoDescribeRoute(
+        Description('Remove user from access control list for a tale.')
+        .modelParam('id', model='tale', plugin='wholetale', level=AccessType.READ)
+        .errorResponse('ID was invalid.')
+        .errorResponse('Access was denied for the tale.', 403)
+    )
+    def relinquishTaleAccess(self, tale):
+        user = self.getCurrentUser()
+        self._model.setUserAccess(tale, user, None, save=True)
+        cherrypy.response.status = 204
```

Comment:
I'm trying to update your `corere-mock` project to test this workflow and girder_client doesn't like the 204. It looks like `sendRestRequest` raises HttpError on anything but 200. https://girder.readthedocs.io/en/latest/_modules/girder_client.html#GirderClient.sendRestRequest

This is only an issue since we're encouraging use of girder_client for integration.